### PR TITLE
GTW-9533: Update descriptions for purchase_country

### PR DESCRIPTION
**Pull request by trev-davies-cko**
# Description of changes in PR

[//]: # 'Please add your description here'

Modifies the description for purchase_country in Request / Capture / GET payments.

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Review by Enrica-Mameli-cko — APPROVED** [review]

File: nas_spec/components/schemas/Payments/CaptureRequestProcessingSettings.yaml [hunk 1]
```diff
@@ -45,4 +45,8 @@ properties:
     type: array
     description: The shipping information for the payment capture. Maximum 500 items.
     items:
-      $ref: '#/components/schemas/ShippingInfo'
\ No newline at end of file
+      $ref: '#/components/schemas/ShippingInfo'
+  purchase_country:
+    type: string
+    description: The country where the goods are purchased / seller is based. ISO 3166 alpha-2 purchase country code.
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/Payments/CaptureRequestProcessingSettings.yaml:51]:
```suggestion
    description: The country where the goods are purchased or the seller is based. ISO 3166 alpha-2 purchase country code.
```

File: nas_spec/components/schemas/Payments/PaymentRequestProcessingSettings.yaml [hunk 2]
```diff
@@ -229,7 +229,7 @@ properties:
                 type: string
   purchase_country:
     type: string
-    description: ISO 3166 alpha-2 purchase country code.
+    description: The country where the goods are purchased / seller is based. ISO 3166 alpha-2 purchase country code.
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/Payments/PaymentRequestProcessingSettings.yaml:232]:
```suggestion
    description: The country where the goods are purchased or the seller is based. ISO 3166 alpha-2 purchase country code.
```

File: nas_spec/components/schemas/Payments/ProcessingData.yaml [hunk 3]
```diff
@@ -27,7 +27,8 @@ properties:
     example: 1000
   purchase_country:
     type: string
-    description: 'The purchase country of the customer, as an ISO 3166 alpha-2 country code.'
+    description: The country where the goods are purchased / seller is based. ISO 3166 alpha-2 purchase country code.
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/Payments/ProcessingData.yaml:30]:
```suggestion
    description: The country where the goods are purchased or the seller is based. ISO 3166 alpha-2 purchase country code.
```

File: nas_spec/changelog.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 | Date       | Description of change                                                                                                 |
 |------------|-----------------------------------------------------------------------------------------------------------------------|
+| 2023/03/15 | Modified the description for purchas_country in Request/Capture/GET payment                                           |
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/changelog.md:5]:
```suggestion
| 2023/03/15 | Modified the description for `purchase_country` in Request/Capture/GET payment                                           |
```

Comment (eoin-shaughnessy-cko) [reply nas_spec/changelog.md:5]: Why is GET capitalized, but Request and Capture aren't?

File: nas_spec/components/schemas/Payments/CaptureRequestProcessingSettings.yaml (hunk 1 above)
Comment (eoin-shaughnessy-cko) [reply nas_spec/components/schemas/Payments/CaptureRequestProcessingSettings.yaml:51]: Feels like there are two definitions for `purchase_country` here: the country, and the country code. Can we expand the second sentence a bit? 'Use the ISO 3166 alpha-2 purchase country code to...', something like that?

**Review by Enrica-Mameli-cko — APPROVED** [review]

**Review by eoin-shaughnessy-cko — APPROVED** [review]

**Pull request merged**